.gitignore

```diff
@@ -0,0 +1 @@
+back/cmd/openbsrv/openbsrv
```

Comment:
This is particular to your workflow and should be in your own global ignore config. Since this name is so specific to this project, it is useful to have a pattern that makes management clean/easy. For example, when I build, the output artifact goes into a subdirectory called "build". That way I can ignore all subdirs named build and not worry about project names.

Reply:
.gitnore was deleted and binary was moved to a build folder which is ignored by globa ignore config.